### ISSUE TITLE
route53_zone: unsupported parameter for module: aws_secret_key

### ISSUE THREAD

**Issue by sgzijl** [ansible/ansible-modules-extras#940]
##### Issue Type:

Bug report.
##### Ansible Version:

2.0 devel
##### Ansible Configuration:

```
inventory      = ec2.py
library        = ./ansible_modules/core:./ansible_modules/extras
roles_path    = roles

action_plugins     = ./ansible_plugins/action_plugins
callback_plugins   = ./ansible_plugins/callback_plugins
connection_plugins = ./ansible_plugins/connection_plugins
lookup_plugins     = ./ansible_plugins/lookup_plugins
vars_plugins       = ./ansible_plugins/vars_plugins
filter_plugins     = ./ansible_plugins/filter_plugins
```
##### Environment:

AWS Route53
##### Summary:

According to http://docs.ansible.com/ansible/route53_zone_module.html the route53_zone module supports the parameters aws_access_key and aws_secret_key, 

When defined, the task fails with:

```
failed: [localhost] => (item=example.tld.) => {"failed": true, "item": "example.tld.", "msg": "unsupported parameter for module: aws_secret_key"
```
##### Steps To Reproduce:

```
- name: Create example.tld
  route53_zone:
    state: present
    zone: example.tld
    aws_access_key: "youraccesskey"
    aws_secret_key: "yoursecretkey"
```
##### Expected Results:

Allow the definition of both parameters, so that cross AWS account actions within the same playbook are possible.
##### Actual Results:

A failure, see summary.

**Comment by steynovich** [comment]
Proposed a fix for this issue. It seems that common AWS parameters (such as aws_access_key, aws_secret_key) and aliases are not included when populating the module arguments.

**Comment by bcoca** [comment]
it seems that it is not loading the shared spec from module_utils/ec2.py

**Comment by bcoca** [comment]
closed via #948
